functions/Find-PasswordStatePassword.ps1

```diff
@@ -120,11 +125,17 @@ Function Find-PasswordStatePassword {
             $headerreason = @{"Reason" = "$reason"}
             $parms = @{ExtraParams = @{"Headers" = $headerreason}}
         }
+        
```

Comment:
```suggestion
```

Reply:
This whitespace

Reply:
that was the issue? 2 tabs?

Reply:
Your suggestion isn't triggering the check?